src/apps/vpn/platforms/windows/daemon/windowsfirewall.cpp

```diff
@@ -21,9 +21,8 @@
 #include <QScopeGuard>
 #include <QtEndian>
 
-#include "../../daemon/interfaceconfig.h"
```

Comment:
Just to confirm, was this removal intentional?

Reply:
yes. it was already included in windowsfirewall.h